GTBitsOfGood/recruitment-portal
# feature(Error Handling): Handle Notion request errors

### Description

Add basic error handling for Notion requests

Closes #5 

- Check the whether the response to the api POST requests is ok
- Show a Snackbar that tells the user the application was not submitted properly

### Type of change
🆕 Feature

### How to Test
- Start application with incorrect Notion .env database credentials and submit an application
- A Snackbar that says "Application failed to submit" should be shown at the bottom left

### Checklist

- [ ] Snackbar should show up for both dev apps and bootie apps
